lib/token.rb

```diff
@@ -0,0 +1,5 @@
+class Token
```

Comment:
I think we should document this a bit, it's not clear why we are using this token. Maybe it needs some documentation here or on the README.md file? like a `Heroku Review Apps Login Proxy` explaining how this works

also, I think it would be better to remove the `algorithm` param, I understand the idea of having flexibility there, but the actual code is not using it and it will be confusing for people reading the code, it's not clear why the param exists when it's only used in the tests, it adds more complexity that is not currently needed. I would go with any hash function for now and only add this configuration if really required in the future